### PR TITLE
questions and answers components working

### PR DESCRIPTION
**Pull request by sunkim0330**
I can render questions and answers using API requests. 
Didn't add any click event handlers yet.
@AlexArms @kate-darby @chrimack Anyone wants to review mine?

### CODE REVIEW

File: client/src/Q&As/Search.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import axios from 'axios';
+
+const Search = () => {
+  return (
+    <div>
+      <input type="text" placeholder="Have a quetion? Search for answers..."/>
```
Comment (chrimack) [inline client/src/Q&As/Search.jsx:7]: typo here in question

File: client/src/Q&As/Answers.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          return setAnswer(response.data.results);
+        })
+        .catch(() => {
+          console.log('cant get request from question API');
+        });
+    }
+  }, [questions]);
+
+  //load up to 2 answers on the page
+  //need to figure out how to hide rest of answers and show them when I click 'load more answers'
```
Comment (chrimack) [inline client/src/Q&As/Answers.jsx:21]: I had to do the same thing. Let me know if you want some tips!

**Review by chrimack — COMMENTED** [review]
These looks pretty good so far! Nice work.

File: client/src/Q&As/Search.jsx (hunk 1 above)
Comment (sunkim0330) [reply client/src/Q&As/Search.jsx:7]: Ah thank you! just fixed it!

File: client/src/Q&As/Answers.jsx (hunk 2 above)
Comment (sunkim0330) [reply client/src/Q&As/Answers.jsx:21]: oh awesome I'll let you know if I need some hints

**Pull request merged**